src/components/forms/text-input/text-input.tsx

```diff
@@ -66,6 +71,7 @@ export const TextInput = (
       <Input
         type={'text'}
         aria-required={true}
+        aria-invalid={Boolean(props.errors?.[props.name])}
         $hasError={Boolean(errorMessage)}
         disabled={formState.isSubmitting}
         id={props.name}
```

Comment:
Here is another suggestion, you have to know yourself whether you want to implement it: There is also the [issue](https://github.com/satellytes/satellytes.com/issues/802#:~:text=Form%20Fields%3A%20aria%2Ddescribedby%20and%20aria%2Dlabeledby%20attributes%20should%20be%20connected%20by%20ID%20to%20an%20element%20that%20either%20has%20text%20description%20or%20an%20%22aria%2Dlabel%22%20attribute.) that the relevant elements of the entire input elements should be linked by id. I think this issue fits perfectly with the addition of the aria-invalid attribute, as the user then not only knows that the field is invalid, but also why it is invalid. Feel free to add this as part of this PR if you also think it belongs together. Here is a diff as an inspiration how the implementation could look like, but it needs to be tested again. 

```diff
diff --git a/src/components/forms/text-input/text-input.tsx b/src/components/forms/text-input/text-input.tsx
--- a/src/components/forms/text-input/text-input.tsx
+++ b/src/components/forms/text-input/text-input.tsx
@@ -67,13 +67,19 @@
         type={'text'}
         aria-required={true}
         $hasError={Boolean(errorMessage)}
+        aria-invalid={Boolean(errorMessage)}
         disabled={formState.isSubmitting}
         id={props.name}
         placeholder={props.placeholder}
         {...field}
         value={(field?.value as string) || ''}
+        aria-describedby={errorMessage ? `${props.name}-error` : undefined}
       />
-      {errorMessage && <StyledErrorMessage>{errorMessage}</StyledErrorMessage>}
+      {errorMessage && (
+        <StyledErrorMessage id={`${props.name}-error`}>
+          {errorMessage}
+        </StyledErrorMessage>
+      )}
     </div>
   );
 };

```